src/git/tag_source_test.go

```diff
@@ -151,14 +214,16 @@ func TestRepoTagsSource_LastVersionHash(t *testing.T) {
 		{
 			name: "Matching_Leading_v",
 			tagOpts: []git.TagOptionFunc{
-				git.TagsMatching("^v"),
+				git.TagsMatchingRegex("^v"),
+				git.TagsMatchingCommits(commitsGetter),
```

Comment:
Why are we adding the `TagsMatchingCommits` opt on this test? It was passing before and we haven't changed the test requirements.

Reply:
I restored the old tests, there was no need for it